ActionSheet.android.js

```diff
@@ -87,19 +96,24 @@ class ActionGroup extends React.Component {
     );
 
     for (let i = startIndex; i < startIndex + length; i++) {
-      let color = '#212121';
-      if (i === destructiveButtonIndex) {
-        color = '#d32f2f';
-      }
-
+      const defaultColor = tintColor
+        ? tintColor
+        : (textStyle || {}).color || BLACK_87PC_TRANSPARENT;
```

Comment:
can you set the defaultProp of a textStyle to an empty object so we don't have to fallback to that here?

Reply:
OK I remember now why I couldn't do it the way you suggest. The parameters for showing an ActionSheet are as follows:

```
showActionSheetWithOptions(
    options: ActionSheetOptions,
    onSelect: (i: number) => void,
    onAnimateOut: () => void
  ) {
```

So I did try to add it to defaultProps, but it just crashes saying textStyle is null. In the render function for ActionSheet, it automatically always uses the passed in options object, which I can't default since that entire object is user-supplied.

Reply:
And I just noticed that my defaultProp for `tintIcons` that I set to true doesn't work for the same reason. So it'll need to default to false/undefined.

Reply:
derp. I'm dumb. I can still do it, just not using defaultProps. 😅 

`textStyle={options.textStyle || {}}`